ci-operator/config/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-main.yaml

```diff
@@ -47,6 +47,27 @@ tests:
   commands: make test-unit
   container:
     from: src
+- as: ci-e2e
+  steps:
+    cluster_profile: aws
+    env:
+      OO_CHANNEL: alpha
+      OO_INSTALL_NAMESPACE: numaresources-operator
+      OO_PACKAGE: numaresources-operator
+      OO_TARGET_NAMESPACES: '!all'
+    test:
+    - as: e2e-test
+      cli: latest
+      commands: make test-e2e
+      dependencies:
+      - env: OO_INDEX
```

Comment:
I think we will need to add some additional ENV variables, see - https://github.com/openshift-kni/numaresources-operator/blob/939709fee7a1c4b58463cb66e353f723139c2c90/.github/workflows/e2e.yml#L21

Reply:
added:
      - env: E2E_NAMESPACE_NAME
        name: numaresources
      - env: E2E_TOPOLOGY_MANAGER_POLICY
        name: SingleNUMANodeContainerLevel

Reply:
These env variables are used for other purposes: 
https://docs.ci.openshift.org/docs/architecture/ci-operator/#referring-to-images-in-tests 

You can remove them.
I added them here: